hydra-cluster/src/Hydra/Cluster/Faucet.hs

```diff
@@ -55,6 +62,7 @@ seedFromFaucet ::
   Tracer IO FaucetLog ->
   IO UTxO
 seedFromFaucet backend receivingVerificationKey val tracer = do
+  delayBF backend
```

Comment:
same here, if seedFromFaucetWithMinting takes the backend, dont u think the delay should go in there?

wdyt?